src/monattack.cpp

```diff
@@ -5801,8 +5801,12 @@ bool mattack::doot( monster *z )
 
 bool mattack::dodge_check( monster *z, Creature *target )
 {
+    const float dodge_ability = target->get_dodge();
+    if( dodge_ability > 0.0 ) {
```

Comment:
```suggestion
    if( dodge_ability > 0.8 ) {
```
Increased to 0.8 based on earlier discussions, to avoid players trying when there's no reasonable chance.
There is room here to estimate success later on and use that instead.